openedx/core/lib/gating/api.py

```diff
@@ -299,3 +304,112 @@ def get_gated_content(course, user):
                 {'id': user.id}
             )
         ]
+
+
+def is_prereq_met(content_id, user_id, recalc_on_unmet=False):
```

Comment:
Suggestion: rename this function so it's clear that it's more costly than just looking up from SQL or a property.  Maybe something like: `compute_is_prereq_met`.

Reply:
@nasthagiri good suggestion, I will change that.